scripts/setup-operators.sh

```diff
@@ -28,9 +28,9 @@ install_etcd_operator(){
   kind: Subscription
   metadata:
     name: etcd
-    namespace: ${OPERATOR_HUB_PROJECT}
+    namespace: openshift-operators
```

Comment:
Earlier, etcd operator was being installed in `CI_OPERATOR_HUB_PROJECT` and mongodb operator in `openshift-operators`. The way this was done made etcd operator available only in the `CI_OPERATOR_HUB_PROJECT` namespace while mongodb operator was available across the cluster.

Now, I have modified the code to install etcd operator in across the cluster as well. The way this works is, we install it in `openshift-operators` namespace using `clusterwide-alpha` channel. This makes sure that the installed operator is available across all the namespaces on the cluster. Serves the purpose of both installing an operator in single namespce as well as across the cluster.